app/grandchallenge/core/validators.py

```diff
@@ -161,7 +161,9 @@ def __call__(self, value):
         try:
             validate(value, self.schema, registry=self.registry)
         except JSONValidationError as e:
-            raise ValidationError(f"JSON does not fulfill schema: {e}")
+            raise ValidationError(
+                f"JSON does not fulfill schema: instance {e.message.replace(str(e.instance), '')}"
```

Comment:
This is not ideal since the format of the message can change any time, but there is currently no better way (that I can think of at least).